src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -97,20 +125,27 @@ export class HeaderFilter<T extends Slick.SlickData> {
             this.setButtonImage($el, column.filterValues?.length > 0);
         }
 
-        $el.on("click", async (e: JQuery.ClickEvent) => {
-            e.stopPropagation();
-            e.preventDefault();
-            this.showFilter($el[0]);
-        });
+        const elDivElement = $el.get(0);
+        if (elDivElement) {
+            this._eventManager.addEventListener(
+                elDivElement,
+                "click",
```

Comment:
extract to const (and elsewhere)

Reply:
@Benjin I'm not quite sure what you mean for this one, could you clarify?